bin/server

```diff
@@ -58,8 +58,6 @@ Object.keys(clusterEnv).forEach(k => {
     clusterEnv[k] = String(clusterEnv[k]);
 });
 
-// Flag to allow external join
-const { externalJoin } = config.get('build');
```

Comment:
can you remove from config files as well?
https://github.com/screwdriver-cd/screwdriver/blob/new/config/custom-environment-variables.yaml#L406
https://github.com/screwdriver-cd/screwdriver/blob/new/config/default.yaml#L320